tests/stdlib/tcgi.nim

```diff
@@ -0,0 +1,23 @@
+discard """
+  action: run
+  file: tcgi.nim
+  output: "[Suite] Test cgi module"
+"""
+
+import unittest
+import cgi, strtabs
+
+suite "Test cgi module":
+  const queryString = "foo=bar&фу=бар&checked=✓&list=1,2,3&with_space=text%20with%20space"

```

Comment:
This looks suspiciously close to my PR: https://github.com/nim-lang/Nim/pull/7761

Reply:
Could we just get rid of this functionality in favour of what I implemented?

This implementation doesn't support "foo=bar&foo=tes" for example. I'm not 100% on CGI, but if it's anything like URIs then this should be supported.

Reply:
@dom96 your PR has still the issue with the converter.

Reply:
Yes, I'm aware.

Reply:
I would prefer that PR was fixed rather than this was accepted.

Reply:
Note the the PR that adds the `readData` proc for strings has already been merged, so you should probably remove it in your PR to avoid duplicating functionality.

Reply:
Also, turns out I'm an idiot and the initial PR was unnecessary to start with. I'm using Jester, so to get the form value I can just use `@"key"` syntax 🤦‍♂️